community/QClass_2024/Submissions/HW3/DM_Lisitha_Dissanayaka_HW3_QClass2024.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<p>Hi!</p><p>Seems to have some problem here. </p><p>In the second case, "Optimal energy" should be around : -1.8... as the electronic energy in class. </p><p>check the solution when it's published :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>